### ISSUE TITLE
Clarify Chocolatey GUI WCAG status, suggesting users make use of Chocolatey CLI

### ISSUE THREAD

**Issue by mkevenaar** [chocolatey/ChocolateyGUI#645]
To improve the accessibility when using screen-readers, the software should be tested using screen-readers

Examples:
- [Jaws](https://www.freedomscientific.com/Products/Blindness/JAWS)

(more will be added later)

@molaachir Could you add extra comments to this issue, if you have more feedback?

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/868h71zkt) by [Unito](https://www.unito.io)

**Comment by molaachir** [comment]
From some basic tests:

1. After passing the search box, and two selection boxes while tabbing through the GUI, you reach 3 buttons and 2 radio buttons that are inaccessible. The screen reader indicates that there is a button or radio button, but they are not labeled. The two selection boxes that you pass on the way there are accessible. They read "Match word exactly" and "Show only packages with updates".
2. The table containing the installed packages is accessible, but if there are no updates, the screen reader reads out what I think is the table ID. Example: "Item: ChocolateyGui.ViewModels.Items.PackageViewModel, Column Display Index: 3", naturally, this only occurs in the updates column, the rest is read out correctly. It should say something like "no updates available for this package".
3. In applications such as Outlook, Word and Google Chrome, the screen reader can display a list of application specific shortcuts when pressing a screen reader specific key combo. i'm not entirely sure that the applications themselves publish these hot keys to the screen reader somehow, or that the screen reader developers added this themselves, perhaps worth investigating. It would be good to be able to bring up a list of shortcuts for reference.
4. There is no information about any shortcuts with regards to the Chocolatey GUI either on the web or in the application itself. Perhaps a dedicated page for this on the site would be a good idea.
5. i don't know about any shortcuts, but i do have a couple of suggestions:
 a. Put focus on the Search field (Ctrl+L or Ctrl+E)
 b. Put focus on the installed packages table (Ctrl+I)
 c. Put focus on the sources list (Ctrl+S)
 d. pin the package focus is currently on (Alt+P)
 e. UnPin package focus is currently on (Alt+U)
 f. Install package focus is currently on (Alt+I)
 g. Uninstall package focus is currently on (Alt+R)
And so on, and so forth.
6. When focus is on an installed package, and you bring up the context menu to pin the package (or perform any other action), and you press enter to perform the action. Focus is lost. you have to navigate back to the table. This makes performing multiple actions unnecessarily time consuming.
7. Introduce audio notifications for success or failiure of actions. Navigating to the console all the time to see if an action was successful can be very time consuming. Introduce little, subtile noises to indecate an action performed successfully or an error occurred. Ideally, you'd have accessibility settings, that allow people to choose if they want notifications sent to the screen reader (like "Package pinned", or "Package installed/uninstalled"), or subtile noises, with the ability to play samples of those noises.

I will update this issue with more findings as i come across them. Feel free to ask questions.

FYI: These tests where done with Jaws, I will test with NVDA as well, in the near future.

**Comment by gep13** [comment]
@molaachir thank you so much for taking the time to add this information to the issue!  I very much appreciate it!

**Comment by alexaveldanez** [comment]
To clarify Chocolatey GUI was not tested with screen readers as part of this issue, instead the documentation for Chocolatey GUI was updated to say that Chocolatey GUI isn't WCAG compliant, with a recommendation that anyone who needs WCAG compliance should use Chocolatey CLI.
